app-backend/api/src/main/scala/user/Routes.scala

```diff
@@ -42,6 +51,11 @@ trait UserRoutes extends Authentication
       pathEndOrSingleSlash {
         get { getUserByEncodedAuthId(authIdEncoded) } ~
         put { updateUserByEncodedAuthId(authIdEncoded) }
+      } ~
+      pathPrefix("dropbox-setup") {
```

Comment:
It makes sense to me for everything related to users / user management / users integrating their other apps to be off of `users` -- all of them are going to modify some `user` in the database, so that doesn't seem very confusing to me.